auth/credentials.py

```diff
@@ -26,4 +26,16 @@
         lambda s: s.split(':', 1),
         allowed_services
     )
-)
\ No newline at end of file
+)
+
+# Allowed extentsions
```

Comment:
This is a list of all enabled extensions, so the word 'allowed' is wrong here.

Also 'alias' is not needed here.